sample_client/build.gradle

```diff
@@ -9,7 +9,7 @@ dependencies {
 }
 
 android {
-    compileSdkVersion 25
+    compileSdkVersion 26
     buildToolsVersion '25.0.0'
```

Comment:
shoudl then always use 26.0.2

Reply:
https://developer.android.com/studio/releases/gradle-plugin.html

> As of gradle 3.0 plugin: With this update, you no longer need to specify a version for the build tools—the plugin uses the minimum required version by default. So, you can now remove the android.buildToolsVersion property.

Do we want to remove it? Less config options is always nice, or?